src/utils.ts

```diff
@@ -0,0 +1,13 @@
+import _ from 'lodash';
+
+export default class Utils {
+  //Converts an object to a query string, ignores properties with undefined/null values
+  static getQueryString(obj: any) {
```

Comment:
If this works for all query parameters you're a god. I'm afraid this can break very quickly, not to block now -- but a little TODO to clean this up would be nice.